docker/web_nginx.conf

```
@@ -0,0 +1,93 @@
+
+## Gio and Sam
+# added an haproxy to better balance the requests
+# between the webnodes
+# http://www.openlibrary.org/admin?stats
+#
+#
+# old conf (requests balanced by nginx)
+#upstream webnodes {
+#    server ol-web1.us.archive.org:8080;
+#    server ol-web2.us.archive.org:8080;
+#}
+#
+# using haproxy
+upstream webnodes {
+    server web_haproxy:7072;
+}
+
+server {
+    listen       80;
+    listen      443;
+    server_name  openlibrary.org;
+
+    include /run/secrets/petabox_seed;
+
```

Fix for the Chrome 85 referrer issue (thanks @jbuckner !)
```suggestion

    # Set the referrer policy so browsers send referrers to our own servers
    # In July 2020, Chrome changed its default referrer policy so any cross-origin
    # requests only sent the root referrer `/`. Since openlibrary.org
    # has a different "origin" than analytics.archive.org, the full referrer
    # path is not sent. This changes the behavior back to the pre-July 2020 change.
    add_header Referrer-Policy "no-referrer-when-downgrade";

```